### ISSUE TITLE
Cant choose items from character inventory or hand during building via building menu or vehicle construction 

### ISSUE THREAD

**Issue by KHeket** [CleverRaven/Cataclysm-DDA#71062]
### Describe the bug

 Previously, if you are welding in hands, as example, car battery and truing to place it at the floor via building menu or install in the car, and there is few batteries nearby - you will have two options in pop up menu:
1) Take item from person inventory
2) Take items from surroundings 
But now this options was removed by someone, and it make problems
For example - you have few car batteries around, some of which is damaged and have draining effect - and when you are trying to place it down at the ground - there will be chosen random battery, so you need firstly move draining batteries out from your range and only then start building

### Attach save file

N/A

### Steps to reproduce

Make character
Spawn two car batteries
Wield one in hands
Try to place down
See no way to choose, which one will be place, and no option to choose items from hands

### Expected behavior

To have options not be removed accidentally by someone

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3930 (22H2)
- Game Version: f2c8c58 [64-bit]
- Graphics Version: Tiles
- Game Language: Русский <color_dark_gray>(92,7%)</color> [ru]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by pjf** [comment]
/confirmed as of `cdda-experimental-2024-01-21-0750`.

In particular, if I start a vehicle construction and there are two different frames available, both get consumed and instead of a vehicle I get a stack backtrace:


<details>
<summary>Click to expand backtrace</summary>

```
11:24:24.580 ERROR : (error message will follow backtrace)
    ./cataclysm-tiles(debug_write_backtrace(std::ostream&)+0x25) [0x1453f7e]
    ./cataclysm-tiles(DebugLog(DebugLevel, DebugClass)+0x281) [0x1453539]
    ./cataclysm-tiles(realDebugmsg(char const*, char const*, char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)+0xa0) [0x1452e0a]
    ./cataclysm-tiles() [0x13c09ed]
    ./cataclysm-tiles(complete_construction(Character*)+0xbf1) [0x13bcbca]
    ./cataclysm-tiles(activity_handlers::build_do_turn(player_activity*, Character*)+0x1bf) [0x10ca1e6]
    ./cataclysm-tiles(activity_type::call_do_turn(player_activity*, Character*) const+0x67) [0x110b2df]
    ./cataclysm-tiles(player_activity::do_turn(Character&)+0x469) [0x1f414fb]
    ./cataclysm-tiles(do_turn()+0x499) [0x14e6625]
    ./cataclysm-tiles(main+0x1309) [0x19bbbb9]
    /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7fe0dc50b083]
    ./cataclysm-tiles(_start+0x2e) [0x1040e2e]

    Attempting to repeat stack trace using debug symbols…
    debug_write_backtrace(std::ostream&)
    …/src/debug.cpp:1226
    DebugLog(DebugLevel, DebugClass)
    …/src/debug.cpp:1453
    realDebugmsg(char const*, char const*, char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
    …/src/debug.cpp:514
    construct::done_vehicle(coords::coord_point<tripoint, (coords::origin)5, (coords::scale)0, false> const&, Character&)
    …/src/debug.h:?
    done_vehicle
    …/src/construction.cpp:1425
    complete_construction(Character*)
    …/src/construction.cpp:1186
    activity_handlers::build_do_turn(player_activity*, Character*)
    …/src/activity_handlers.cpp:?
    activity_type::call_do_turn(player_activity*, Character*) const
    …/src/activity_type.cpp:160
    player_activity::do_turn(Character&)
    …/src/string_id.h:239
    do_turn
    …/src/player_activity.cpp:291
    do_turn()
    …/src/do_turn.cpp:?
    main
    …/src/main.cpp:851
    addr2line: DWARF error: section .debug_info is larger than its filesize! (0x93f189 vs 0x530f70)
    __libc_start_main
    ??:?
    _start
    ??:?
Backtrace emission took 2 seconds.
(continued from above) ERROR : src/construction.cpp:1425 [void construct::done_vehicle(const tripoint_bub_ms &, Character &)] constructing failed: components size expected 1 actual 2
```

</details>

**Comment by RenechCDDA** [comment]
Resolved by #72201, per https://github.com/CleverRaven/Cataclysm-DDA/issues/71080#issuecomment-2107359587
